lib/tasks_private/release.rake

```diff
@@ -3,7 +3,6 @@ Rake::Task[:release].clear
 desc "Release a new project version"
 task :release do
   require 'pathname'
-  require 'yaml'
   require 'more_core_extensions/core_ext/hash/nested'
```

Comment:
@Fryguy with your new approach, I think you can also remove `require 'more_core_extensions/core_ext/hash/nested'` as it was being used in `content.store_path` method

Reply:
Nice.